src/pages/LandingPage/components/EventList/styled.ts

```diff
@@ -14,6 +14,7 @@ export const GridList = styled.ul`
   grid-gap: ${usingTypography((t) => t.scaleSpacing(5))}px;
   justify-content: center;
   margin: auto;
+  margin-bottom: 20px;
```

Comment:
See my other comment regarding how this ideally should be moving into an ancestor element that's shared by all pages.